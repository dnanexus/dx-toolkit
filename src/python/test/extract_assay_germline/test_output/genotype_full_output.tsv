sample_id	allele_id	locus_id	chr	pos	ref	alt	genotype_type
