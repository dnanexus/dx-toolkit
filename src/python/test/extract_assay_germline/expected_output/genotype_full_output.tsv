	sample_id	a_id	locus_id	chr	pos	ref	alt	type
4	sample_1_0	18_47360_C_CT	18_47360_C_T	18	47360	C	CT	het-alt
